README.md

```diff
@@ -46,26 +67,61 @@ Defining them is part of the set up of your project.
-- `FRONTEND_PORT` = _This is the local Port on which your website (client-side) would be reachable (frontend)_
-- `BACKEND_PORT` = _This is the local Port on which your server would be reachable (backend)_
+- `FRONTEND_PORT` = _This is the local Port on which your website (client-side) would be reachable_
+- `BACKEND_PORT` = _This is the local Port on which your server would be reachable_
 - `DAPP_ACCOUNT_MNEMONIC` = _This is the mnemonic of the Kilt account paying for all transactions_
 - `DAPP_DID_MNEMONIC` = _This is the mnemonic of the Kilt DID that identifies your dApp_
 - `DAPP_DID_URI` = _This is the URI of the Kilt DID that identifies your dApp_
 - `DAPP_NAME` = _This should be a custom name for your dApp_
 - `JWT_SIGNER_SECRET` = _This is secret key (string) that signs the Json-Web-Tokens before saving them in the Cookies_
 
-There is a script to facilitate the generation of the environment variables that are needed to set up your decentralized App (dApp).
-This script is called `./scripts/genesisEnvironmentVariables.ts`.
-You can execute it by running `yarn environment`.
+### How do I get mine? 
+You have three different options: 
+
+1. Using `yarn environment`:
+   
+      There is a script to facilitate the generation of the environment variables inside of this repository.
+      This script is called `./scripts/genesisEnvironmentVariables.ts`.
+
+     + You can execute it by running `yarn environment`.
+
+      Setting up your environment with this script is a step by step process.
+      You will need to run the script repeatedly and follow the instructions that it provides, depending on your project's state.
+      After running this script each time, you need to manually copy the output and save it on the `.env`-file . 
+
+3. Usign the [kilt-distillery-cli](https://github.com/KILTprotocol/kilt-distillery-cli)
+   
+      This is a _Command Line Interface_ tool that can help you obtaining this variables and also does other common tasks (unrelated to this project).
+      The distillery uses the same key derivation as this repository, wich means that it is highly compatible. 
+
+5. Without help:
+   
+      If you are a _pro_, you could defined and generate them externally and add them to the `.env`-file. 
+      You would probably have to modify the `generateKeyPairs.ts` files (on _scripts_ and _backend_) to match your key derivation though.  
+
+## Well-Known-DID-Configuration
+
+After having all your enviorement variables, to build a **Domain Linkage Credential**:
+
++ run: `yarn well-known-config`.
+
+If you want know more about this, check out the [Identity Foundation Documentation](https://identity.foundation/.well-known/resources/did-configuration/). 
+[Our dApp documentation also have a section about this.](https://docs.kilt.io/docs/develop/dApp/well-known-did-config)
+
+## Build 
+After having all your enviorement variables and your well-known-did-configuration: 
```

Comment:
why is this in the documentation?

Reply:
The link to our documentation on the same topic?